### PR TITLE
add implementation for tasmota fw

### PR DESCRIPTION
**Pull request by macpijan**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# RF Sonoff library
+
+Two implemeentations are present here:
+- `sonoff-api-tasmota.robot` - for devices running
+  [Tasmota](https://tasmota.github.io/docs/) firmware
+- `sonoff-api-esphome.robot` - for devices running
+  [ESPHome firmware](https://esphome.io/devices/sonoff.html)
```
Comment (pkubaj) [inline README:7]: Trailing whitespace.

**Comment by pkubaj** [comment]
Is it intentional that there is  `sonoff-api-esphome.robot` and `sonoff-esphome-api.robot` with the same content but slightly different keyword names?

**Comment by pkubaj** [comment]
Also, as I understand it, setting which firmware is to be used is by creating `sonoff-api.robot` symlink, is that right?

File: README (hunk 1 above)
Comment (pkubaj) [inline README]: Trailing whitespace.

**Review by pkubaj — CHANGES_REQUESTED** [review]

**Comment by macpijan** [comment]
> Also, as I understand it, setting which firmware is to be used is by creating sonoff-api.robot symlink, is that right?

Currently it is, since it was simply the easiest.

We are using tasmota only now, so should not be a prolbem.

**Comment by macpijan** [comment]
> Is it intentional that there is sonoff-api-esphome.robot and sonoff-esphome-api.robot with the same content but slightly different keyword names?

Yes it was, especially when we have not migrated yet (we had a mix of different firwmares in the lab). But now we have tasmota only FW in lab.

File: README (hunk 1 above)
Comment (macpijan) [reply README]: Fixed in: 0783b18032d405a556ec684b0b9287b0f9254919

**Comment by macpijan** [comment]
Clarified usage in: 0783b18032d405a556ec684b0b9287b0f9254919

Merging since we only use Tasmota right now.

**Pull request merged**